extensions/arc/src/localizedConstants.ts

```diff
@@ -79,6 +79,7 @@ export const resetAllToDefault = localize('arc.resetAllToDefault', "Reset all to
 export const resetToDefault = localize('arc.resetToDefault', "Reset to default");
 export const troubleshoot = localize('arc.troubleshoot', "Troubleshoot");
 export const clickTheNewSupportRequestButton = localize('arc.clickTheNewSupportRequestButton', "Click the new support request button to file a support request in the Azure Portal.");
+export const supportRequestNote = localize('arc.supportRequestNote', "* Must have resource configuration uploaded to Azure portal to open a support request.");
```

Comment:
Run this by the designers/PMs to make sure they're ok with the format - not sure the * is the best way to do this (normally that would imply a * in the text above that it's referring to)

Just something like `Note that the resource configuration must have been uploaded to Azure first in order to open a support request`

Can we also find out if there's some fallback support request option for users that want support but haven't been able to upload to Azure? It would be nice to have - especially for users that my be having trouble uploading to Azure in the first place!